app/assets/javascripts/add.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
```

Comment:
Why did you use jQuery here? In Angular app we dont use this. Go for Angular features.